lib/mdot/client.rb

```diff
@@ -59,7 +59,10 @@ def submit_order(request_body)
     private
 
     def headers
-      { veteranId: @user.ssn }
+      {
+        VA_VETERAN_ID: @user.ssn.split(//).last(4).join,
```

Comment:
how about `@user.ssn.last(4)`?

Reply:
gotta love that rails magic, tested this in irb and it did not work, but sure enough in the rails console it does. changing it.